docs/network-spec/connection-manager.tex

```diff
@@ -1705,53 +1713,144 @@ \section{Inbound Protocol Governor}
-\textsf{Local} subscript is mapped to identity:
-\begin{lstlisting}
-*'$\Phi$(\textsf{a\textsubscript{Local}}) = \textsf{id}'*
-*'$\Phi$(\DemotedToColdAnyRem:\InboundIdleStateAny{}$\rightarrow$\TerminatingState)'*
-  = *'$\text{\CommitRemote}\circ\text{\TimeoutExpired}$'*
-*'$\Phi$(\DemotedToColdAnyRem:\DuplexState{}$\rightarrow$\OutboundStateDupTau)'*
-  = *'\RemoteToCold'*
-*'$\Phi$(\PromotedToWarmAnyRem) = \AwakeRemote'*
-*'$\Phi$(\TimeoutExpired)       = \TimeoutExpired'*
-*'$\Phi$(\AwakeAnyRem)          = \AwakeRemote'*
-\end{lstlisting}
+\subsection{States}
+
+The state of the inbound governor are similar to the outbound governor, but
+there are crucial differences.
+
+\subsubsection{\RemoteCold}
+The remote cold state segnifies that the remote peer is not using the
+connection, however the only reason why the inbound governor needs to track
+that connection is because the outbound side of this connection is used.  The
+inbound governor will wait until any of the responder mini-protocols wakes up
+(\AwakeRemote{}) or the mux will be shutdown (\MuxTerminated{}).
+
+\subsubsection{\RemoteIdle}
+The \RemoteIdle{} state is an intial state of each connection
+(\NewConnection{}).  An active connection will become \RemoteIdle{} once the
+inbound governor will detect that all responder mini-protocols terminated
+(\WaitIdleRemote{}).  When a connection enters this state, an idle timeout is
+started.  If no activity will be detected on the responders, the connection
+will either be closed by the connection manager and forgotten by the inbound
+governor, or progress to the \RemoteCold{} state.  This depends whether the
+connection is used or not by the outbound side.
+
+\subsubsection{\RemoteWarm}
+A connection enters \RemoteWarm{} state once any of the mini-protocols starts
+to operate.  Once all hot mini-protocols started the state will transition to
+\RemoteHot{}.  Note that this is slighly different than the notion of a \warm{}
+peer, for which all \established{} and \warm{} mini-protocols are active, but
+\hot{} ones are idle.
+
+\subsubsection{\RemoteHot}
+A connection enters \RemoteHot{} transition once all hot protocol started, if
+any of them terminates the connection will be put in \RemoteWarm{}.
+
+\subsection{Transitions}
+
+\subsubsection{\NewConnection}
+Inbound and outbound duplex connections are passed to the inbound governor.
+They are then put in \RemoteIdle{} state.
+
+\subsubsection{\CommitRemote}
+Once the \RemoteIdle{} timeout expires the inbound governor will call
+\texttt{unregisterInboundConnection}.  Depending on the returned value the
+connection will either be forgotten or kept in \RemoteCold{} state.
+
+\subsubsection{\AwakeRemote}
+While a connection was put in \RemoteIdle{} state it is possible that the
+remote end will start using it.  When the inbound governor will detect that any
+of the responders is active it will put that connection in \RemoteWarm{} state.
```

Comment:
When the inbound governor will detect that any of the responders is active -> When the inbound governor detects that any of the responders became active